interface-definitions/dhcp-server.xml.in

```diff
@@ -373,6 +373,20 @@
                       </leafNode>
                     </children>
                   </tagNode >
+                  <leafNode name="ipv6-only-preferred">
+                    <properties>
+                      <help>Signal IPv6-only-capable hosts to disable the IPv4 stack on the host interface (RFC 8925)</help>
+                      <valueHelp>
+                        <format>u32</format>
+                        <description>The number of seconds the client should disable DHCPv4 for</description>
+                      </valueHelp>
+                      <constraint>
+                        <validator name="numeric" argument="--range 0-4294967295"/>
+                      </constraint>
+                      <constraintErrorMessage>IPv6-only wait time must be between 0 and 4294967295 (49 days)</constraintErrorMessage>
+                    </properties>
+                    <defaultValue>0</defaultValue>
```

Comment:
Why do we need a default value here? Simply using no default value should disable this feature.

Reply:
Removed.